packages/cloud-cognitive/src/components/CreateTearsheet/CreateTearsheet.mdx

```diff
@@ -0,0 +1,28 @@
+import { Story, Props, Preview } from '@storybook/addon-docs/blocks';
+import { pkg } from '../../settings';
+import { getStorybookSlug } from '../../../config';
+import { CreateTearsheet } from '.';
+
+# CreateTearsheet
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Example usage](#example-usage)
+- [Component API](#component-api)
+
+## Overview
+
+This component is an example
```

Comment:
Is it an example?

Reply:
Yeah I need to update the doc file here 👍